src/components/List.js

```diff
@@ -4,15 +4,29 @@ import Page from "./Page";
 
 class List extends React.Component {
 
+    roundedCorners(NUMCOLUMNS,COLUMNSIZE,i) {
+        if (i % (NUMCOLUMNS*COLUMNSIZE) === COLUMNSIZE - 1) {
+            return "round-bl";
+        } else if (i % (NUMCOLUMNS*COLUMNSIZE) === (NUMCOLUMNS*COLUMNSIZE) - (COLUMNSIZE)) {
+            return "round-tr";
+        }
+    }
+
     render() {
         const NUMCOLUMNS = this.props.columns;
-        const COLUMNSIZE = 13;
+        const COLUMNSIZE = 12;
 
         const id = this.props.id || "listCarousel";
-        const listItems = this.props.calls.map(call => <Call {...call} key={call.name} onClick={() => this.props.onClick(call.name)} />);
+
+        var listItems = [];
+        for (var i = 0; i < this.props.calls.length; i++) {
+            const call = this.props.calls[i];
+            listItems.push(<Call {...call} key={call.name} rounded={this.roundedCorners(NUMCOLUMNS,COLUMNSIZE,i)} onClick={() => this.props.onClick(call.name)} />)
+        }
         while (listItems.length % (NUMCOLUMNS*COLUMNSIZE) !== 0 || listItems.length === 0) {
-            listItems.push(<Call empty={true} name="~" group={0} key={`${id}, ${listItems.length}`} />)
+            listItems.push(<Call empty={true} rounded={this.roundedCorners(NUMCOLUMNS,COLUMNSIZE,listItems.length)} name="~" group={0} key={`${id}, ${listItems.length}`} />)
```

Comment:
Why do you use `~` as a name for the empty ones?
Also I would compute a variable called `hasRoundedCorners` in the first line of this while loop and then pass it into the `rounded={hasRoundedCorners}`. It's easier to read.

Reply:
I'm calling the variable roundedCorners rather than hasRoundedCorners because at max each call has a single rounded corner